### PR TITLE
Projects modal popup

### PR DESCRIPTION
**Pull request by mgmediaweb**
# Features

- Create an HTML for the curtain and the float window popup.
- Use flexbox to center de window into the center of the screen (I use the same code for mobile and desktop version).
- Create a Javascript Object for the information of the projects.
- Use JS function to manage the indicator and show the modified information (and image) before opening the window.
- Use CSS transition to apply some animation to show/hide the window.
- All checks passed.

### CODE REVIEW

File: Assets/js/scripts.js [hunk 1]
```diff
@@ -62,6 +62,16 @@ const goto = (section) => {
   menuClose();
 };
 
+const modalClose = (opc = false) => {
```
Comment (ShahierNashaat) [inline Assets/js/scripts.js:65]: - [ ] Please add the `link to live version` and `link to the project source code` attributes in each project object in the projects array.
- [ ] Also update the href value in the `see live` and `see source` links with values of the `link to live version` and `link to the project source code` attributes.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @mgmediaweb and @diegoh40 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Assets/js/scripts.js (hunk 1 above)
Comment (mgmediaweb) [reply Assets/js/scripts.js:79]: in the new commit, I repair this issue, thanks for the feedback

**Review by ShahierNashaat — APPROVED** [review]
Hi @mgmediaweb and @diegoh40 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**